src/module/PrcaOrchestrator.ts

```diff
@@ -111,4 +116,15 @@ export class PrcaOrchestrator {
         return result;
     }
 
+    /* Static methods */
+
+    public static CreatePrcaOrchestrator(logger: ILogger, collectionUrl: string, token: string, repositoryId: string, pullRequestId: number): PrcaOrchestrator {
```

Comment:
It seems to me that this method is going to be used by the product to create the orhcestrator. In this case it should take the role of the ctor. Then change the original ctor to be a static creator helper for test purposes.